# Generated by Django 5.1.1 on 2025-04-04 09:40

import datetime
import django.db.models.deletion
from django.conf import settings
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('core', '0046_rename_distance_meters_fitnessactivity_distance_km'),
    ]

    operations = [
        migrations.RemoveField(
            model_name='medicationreminder',
            name='reminder_time',
        ),
        migrations.RemoveField(
            model_name='medicationreminder',
            name='status',
        ),
        migrations.AddField(
            model_name='medicationreminder',
            name='created_at',
            field=models.DateTimeField(auto_now_add=True, default=datetime.datetime(2025, 4, 4, 9, 39, 26, 834735, tzinfo=datetime.timezone.utc)),
            preserve_default=False,
        ),
        migrations.AddField(
            model_name='medicationreminder',
            name='day_of_week',
            field=models.IntegerField(default=1),
            preserve_default=False,
        ),
        migrations.AddField(
            model_name='medicationreminder',
            name='hour',
            field=models.IntegerField(default=0),
        ),
        migrations.AddField(
            model_name='medicationreminder',
            name='minute',
            field=models.IntegerField(default=0),
        ),
        migrations.AddField(
            model_name='medicationreminder',
            name='repeat_weeks',
            field=models.IntegerField(default=4),
        ),
        migrations.AddField(
            model_name='medicationreminder',
            name='updated_at',
            field=models.DateTimeField(auto_now=True),
        ),
        migrations.AlterField(
            model_name='medicationreminder',
            name='user',
            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL),
        ),
    ]
